.github/workflows/pants.yaml

```diff
@@ -0,0 +1,62 @@
+---
+name: Validate Pants Metadata
+
+on:
+  # temporarily only allow manual runs until we have BUILD files and lockfiles
+  workflow_dispatch:
+  #push:
+  #  branches:
+  #    # only on merges to master branch
+  #    - master
+  #    # and version branches, which only include minor versions (eg: v3.4)
+  #    - v[0-9]+.[0-9]+
+  #  tags:
+  #    # also version tags, which include bugfix releases (eg: v3.4.0)
+  #    - v[0-9]+.[0-9]+.[0-9]+
+  #pull_request:
+  #  type: [opened, reopened, edited]
+  #  branches:
+  #    # Only for PRs targeting those branches
+  #    - master
+  #    - v[0-9]+.[0-9]+
```

Comment:
We will uncomment this and remove `workflow_dispatch` when we add the BUILD files.